lib/models/repositoryState.js

```diff
@@ -1,6 +1,7 @@
 var _ = require('lodash');
 var Immutable = require('immutable');
 
+var Normalize = require('../utils/normalize');
```

Comment:
`normalize` instead of `Normalize`, as far as I can see. It is not a class constructor.